h2o-core/src/main/java/hex/ScoreKeeper.java

```diff
@@ -126,9 +126,10 @@ else if (m instanceof ModelMetricsMultinomial) {
     RMSE(ConvergenceStrategy.LESS_IS_BETTER, true),
     MAE(ConvergenceStrategy.LESS_IS_BETTER, true),
     RMSLE(ConvergenceStrategy.LESS_IS_BETTER, true),
-    AUC(ConvergenceStrategy.MORE_IS_BETTER, false),
```

Comment:
the lower bound only matters to LESS_IS_BETTER, imho it doesn't matter whether we have true or false there - but obviously AUC is actually lower bound by 0

Reply:
Got it.

Reply:
Can I keep it as true though?  It looks better.

Reply:
yes, no harm there - I agree it does look better